test/ReactiveSocketTest.cpp

```diff
@@ -248,20 +249,27 @@ TEST(ReactiveSocketTest, Destructor) {
   auto serverConn = folly::make_unique<InlineConnection>();
   clientConn->connectTo(*serverConn);
 
+  // TODO: since we don't assert anything, should we just use the StatsPrinter
+  // instead?
```

Comment:
Oh, sorry, glog is fine. As long as we don't use cout/cerr directly. But I would actually add assertions for those stats.